### DISCUSSION TITLE
Need some volunteers to flesh out a PRD with Tristan

### DISCUSSION THREAD

**Discussion by kljensen** [yale-research-job-board/.github#1] [General]
Ahoy Friends, thanks for joining. We need to get a product requirements document (PRD) with a _rough_ sketch of what this app should be. [Tristan](https://som.yale.edu/faculty-research/faculty-directory/tristan-l-botelho) is the major stakeholder. Hoping

1. Some of you will volunteer to flesh this out with him (and potential users)
2. Those volunteers will come up with an interview strategy for him...and also
3. Major personas and user journeys, etc.

For 2 & 3 it would be nice if you had some framework...there's a few different ones floating around.

**Comment by yashswijain** [comment]
I, @yiyanghe1, @komalbagrodia97 will be taking care of this.

**Comment by yashswijain** [comment]
Prof. Tristan has indicated his availability for the week starting Jan 6th. Will update once the time of the kickoff meeting is finalised. 
We will be conducting job board research till then.
